src/main/java/bio/terra/service/common/azure/StorageTableUtils.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.service.common.azure;
+
+import java.util.UUID;
+
+public class StorageTableUtils {
+  private static final String DATASET_TABLE_NAME = "dataset";
+  private static final String FILES_TABLE_NAME = "files";
+
+  /**
+   * Generate a Storage Table name from a UUID
+   *
+   * @param resourceId The datasetId or snapshotId to be used as root of the table name
+   * @param suffix suffix for storage table Name
+   * @return A valid azure storage table name
+   */
+  public static String toTableName(String resourceId, StorageTableNameSuffix suffix) {
+    return "datarepo" + resourceId.replaceAll("-", "") + suffix.getLabel();
+  }
+
+  public static String toTableName(UUID resourceId, StorageTableNameSuffix suffix) {
+    return toTableName(resourceId.toString(), suffix);
+  }
+
+  // TODO - With DR-2127, remove this  method and add case for dataset in toTableName
+  public static String getDatasetTableName() {
```

Comment:
I don't think we need these to have getter methods. Why not just make them `public static final` fields and reference directly?

Reply:
That's a great point - thank you!